SmartTutor/ContentModel/Lectures/KnowledgeNode.cs

```diff
@@ -9,9 +9,12 @@ public class KnowledgeNode
         public KnowledgeNodeType Type { get; private set; }
         public int LectureId { get; private set; }
         public List<LearningObjectSummary> LearningObjectSummaries { get; private set; }
+        public List<KnowledgeNode> Prerequisites;
```

Comment:
Are we missing {get; private set;} here? Also, can we add private to the setter of PrerequisiteOf?